src/main/java/com/tvtracker/enterprise/dao/MediaEntryDAOStub.java

```diff
@@ -41,7 +41,7 @@ public MediaEntry save(MediaEntry mediaEntry) throws Exception {
      * @return List of MediaEntry objects belonging to the given user
      */
     @Override
-    public List<MediaEntry> fetchByUsername(String username) throws Exception {
+    public List<MediaEntry> fetchByUsername(String username) {
```

Comment:
The exception was not used so I got rid of it.

Reply:
Are you certain it won't be used in the future?  This is a stub, which won't throw an exception, but the actual implementation could very well throw an exception (SQLException, FileNotFoundException, etc.)  Unless you're certain that it won't be thrown, I lean toward leaving this in.